openshift/e2e-common.sh

```diff
@@ -192,11 +192,7 @@ EOF
   path: spec.install.spec.deployments.(name==knative-operator).spec.template.spec.volumes[+]
```

Comment:
(can't comment further above) Let's adjust the comment above this to also not be cocnerned about kourier.yaml anymore.

Reply:
Yes, updated the comment.